build.gradle

```diff
@@ -17,6 +21,16 @@ allprojects {
     project.apply from: "${rootDir}/constants.gradle"
 }
 
+subprojects {
+    apply plugin: 'io.spring.dependency-management'
```

Comment:
Spring, dependency management ?

Reply:
Yes, Gradle itself does not support BOMs or something similar. So we use a plugin provided by Spring that allows us to make use of Maven BOMs in Gradle.

Reply:
https://github.com/spring-gradle-plugins/dependency-management-plugin

Enables bom support